webapp/src/app/api/Errors.ts

```diff
@@ -0,0 +1,5 @@
+export class LanguageNotFound extends Error {
```

Comment:
I think this file should be `src/errors.ts` instead. Both the lowercase name, and the location outside of `src/app` which is a special Next.js folder.